package.json

```diff
@@ -38,15 +38,16 @@
     "prepublish": "npm run lint && npm run build && npm run bootstrap",
     "unsafe-bootstrap": "bash bin/bootstrap",
     "visualizer": "npm run watch & bin/ohm-visualizer",
-    "watch": "watchify $npm_package_browserify_options -v"
+    "watch": "bash bin/watch.sh"
   },
   "license": "MIT",
   "author": "Alex Warth <alexwarth@gmail.com> (http://tinlizzie.org/~awarth)",
   "contributors": [
     "Patrick Dubroy <pdubroy@gmail.com>",
     "Tony Garnock-Jones <tonygarnockjones@gmail.com>",
     "Yoshiki Ohshima <Yoshiki.Ohshima@acm.org>",
-    "Marko Röder <m.roeder@photon-software.de>"
+    "Marko Röder <m.roeder@photon-software.de>",
```

Comment:
Sorry, would you mind leaving this out of this PR? Totally happy to give credit to everyone who has contributed, but right now this list is kind of arbitrary...and rather than add only you, I think we should just add everyone who has contributed, or drop this from the package.json. Either way, I'd prefer to handle in a separate PR.